drivers/overlay/overlay.go

```diff
@@ -743,7 +743,11 @@ func (d *Driver) get(id string, disableShifting bool, options graphdriver.MountO
 
 	workDir := path.Join(dir, "work")
 	opts := fmt.Sprintf("lowerdir=%s,upperdir=%s,workdir=%s", strings.Join(absLowers, ":"), diffDir, workDir)
-	if d.options.mountOptions != "" {
+	if options.Options != nil {
+		if len(options.Options) > 0 {
```

Comment:
The length of a `nil` slice is defined as 0, so we don't need to check for `nil` before checking the length.